src/main/java/com/minecolonies/sounds/CitizenSounds.java

```diff
@@ -0,0 +1,82 @@
+package com.minecolonies.sounds;
+
+import com.minecolonies.util.SoundUtils;
+import net.minecraft.util.SoundEvent;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+
+/**
+ * Class used to store the basic citizen sounds.
+ */
+public final class CitizenSounds
+{
+    /**
+     * Chance to say a phrase.
+     */
+    private static final int PHRASE_CHANCE = 50;
+
+    /**
+     * The citizen sound events for the females.
+     */
+    public static final class Female
+    {
+        public static final SoundEvent say = ModSoundEvents.getSoundID("mob.citizen.female.say");
+
+        /**
+         * Private constructor to hide the implicit public one.
+         */
+        private Female()
+        {
+            /*
+             * Intentionally left empty.
+             */
+        }
+    }
+
+    /**
+     * The citizen sound events for the males.
+     */
+    public static final class Male
+    {
+        public static final SoundEvent say = ModSoundEvents.getSoundID("mob.citizen.male.say");
+
+        /**
+         * Private constructor to hide the implicit public one.
+         */
+        private Male()
+        {
+           /*
+            * Intentionally left empty.
+            */
+        }
+    }
+
+    /**
+     * Private constructor to hide the implicit public one.
+     */
+    private CitizenSounds()
+    {
+        /*
+         * Intentionally left empty.
+         */
+    }
+
+    /**
+     * Plays citizen sounds.
+     *
+     * @param worldIn  the world to play the sound in.
+     * @param position the position to play the sound at.
+     * @param isFemale the gender.
+     */
+    public static void playCitizenSounds(World worldIn, BlockPos position, boolean isFemale)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Provide multiple methods instead of using "isFemale" to determine which action to take. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS2301)